test/functional/api/vim_spec.lua

```diff
@@ -1,7 +1,9 @@
 local helpers = require('test.functional.helpers')(after_each)
 local Screen = require('test.functional.ui.screen')
+local lfs = require('lfs')
```

Comment:
lfs is never needed, we should add a lint check for this. Nvim functions are available from `funcs` (line 13 below)